src/connect.spec.lua

```diff
@@ -219,11 +221,11 @@ return function()
 
 	it("should dispatch the action using a table of action creators", function()
 		local mapDispatchToProps = {
-			increment = function()
+			increment = (function()
 				return {
 					type = "increment",
 				}
-			end,
+			end :: any) :: AnyActionCreator,
```

Comment:
Does this explicit cast need a TODO luau comment? I'm curious if this isn't necessary with Alan's subtyping fix that just merged (but luau-LSP would lag on until  this weeks OSS Luau release)?

Reply:
ActionCreator is actually a table with some metatable shenanigans to allow indexing a callable, but we're passing in regular degular functions here (which is fine for these tests). I don't know off the top of me head if Luau can compare function types and callable tables, but I don't think passing a function type into something that expects a callable table to be type safe anyway (although the other way around might work ignoring typeof() edge cases).